src/System.ComponentModel.TypeConverter/src/System/ComponentModel/BackgroundWorker.cs

```diff
@@ -0,0 +1,267 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+
+
+
+
+namespace System.ComponentModel
+{
+    //using System.ComponentModel.Design.Serialization;
+
+    using System.Diagnostics;
+    using System.Security.Permissions;
+    using System.Threading;
+
+    [
+        SRDescription("BackgroundWorker_Desc"),
+        DefaultEvent("DoWork"),
+        HostProtection(SharedState = true)
+    ]
+    public class BackgroundWorker : Component
```

Comment:
Don't we already have BackgroundWorker in System.ComponentModel.EventBasedAsync?